tests/modules/programs/scmpuff/fish.nix

```
@@ -0,0 +1,21 @@
+{ pkgs, lib, ... }: {
+  config = {
+    programs = {
+      scmpuff.enable = true;
+      fish.enable = true;
+    };
+
+    # Needed to avoid error with dummy fish package.
+    xdg.dataFile."fish/home-manager_generated_completions".source =
+      lib.mkForce (builtins.toFile "empty" "");
+
+    test.stubs.fish = { };
```

```suggestion
    test.stubs.fish = { };
    test.stubs.scmpuff = { };
```
Is there anything prevent the scmpuff package from being overriden?

---

Done